modules/dashboard/modules/widgets/sla-slo-sli/base.tf

```diff
@@ -3,7 +3,7 @@ module "base" {
 
   coordinates = var.coordinates
 
-  name = "SLA"
+  name = var.balancer_name != null ? "${var.balancer_name} SLA" : "SLA"
```

Comment:
Is this useful if balancer name is not passed?
Isn't it better to thrown an error when not?